Userland/Libraries/LibJS/Bytecode/Op.cpp

```diff
@@ -1008,6 +1061,24 @@ String Call::to_string_impl(Bytecode::Executable const&) const
     return builder.to_string();
 }
 
+String SuperCall::to_string_impl(Bytecode::Executable const&) const
+{
+    StringBuilder builder;
+    builder.append("SuperCall"sv);
+    if (m_is_synthetic) {
+        builder.append(" arguments:[...acc]"sv);
+    } else if (m_argument_count != 0) {
+        builder.append(" arguments:["sv);
+        for (size_t i = 0; i < m_argument_count; ++i) {
+            builder.appendff("{}", m_arguments[i]);
+            if (i != m_argument_count - 1)
+                builder.append(',');
+        }
+        builder.append(']');
```

Comment:
String::join pls

Reply:
Turns out the compiler error is just not helpful
(It complains about the `incomplete type 'const JS::Bytecode::Register[]'` and not about the missing const you would have gotten after the type decay)